docs/content/AutoGPT/Setups/nogit-setup.md

```diff
@@ -82,4 +82,4 @@ packages and launch AutoGPT.
     ```
 
 If this gives errors, make sure you have a compatible Python version installed. See also
-the [requirements](./installation.md#requirements).
+the [System Requirements](../../../../QUICKSTART.md).
```

Comment:
This doesn't work, see the [preview](https://deploy-preview-6324--auto-gpt-docs.netlify.app/autogpt/setups/nogit-setup/). The link should point to `/autogpt/setup/#requirements` or `../#requirements`.

Reply:
The path `/autogpt/setup/` doestn't contain any requirements file..

Reply:
It's a section on the setup page: https://docs.agpt.co/autogpt/setups/#requirements

Reply:
Two possible solutions:

We should remove the line -  See also the [requirements](https://github.com/Significant-Gravitas/AutoGPT/blob/master/docs/content/AutoGPT/Setups/installation.md#requirements).

or 

Creating and adding requirements.md file to `/autogpt/setup/`

Reply:
> It's a section on the setup page: https://docs.agpt.co/autogpt/setup/#requirements

Okay got it.